x/bank/client/rest/sendtx.go

```diff
@@ -107,6 +107,16 @@ func SendRequestHandlerFn(cdc *wire.Codec, kb keys.Keybase, cliCtx context.CLICo
 			txCtx = newCtx
 		}
 
+		if utils.HasGenerateOnlyArg(r) {
```

Comment:
Can we encapsulate this logic in a common helper function?

Reply:
(also applies to other instances)

Reply:
ACKd and fixed.